client/src/components/forms/FormButton/FormButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from 'react';
+import { Button, Form } from 'antd';
+import type { ButtonProps, FormItemProps } from 'antd';
+import s from './FormButton.module.less';
+
+interface Props
+  extends Omit<ButtonProps, 'name' | 'onReset'>,
+    Omit<FormItemProps, 'children' | 'name' | 'onReset'> {
+  buttonName?: ButtonProps['name'];
+  buttonOnReset?: ButtonProps['onReset'];
+  formItemName?: FormItemProps['name'];
+  formItemOnReset?: FormItemProps['onReset'];
+}
+
+export const FormButton: FC<Props> = ({
+  buttonName,
+  children,
+  buttonOnReset,
+  formItemOnReset,
+  formItemName,
+  htmlType = 'submit',
+  type = 'primary',
+  ...rest
+}) => {
```

Comment:
Funkcja z trzema argumentami to już tłoczno. Tak żeby było super ładnie to pewnie idzie to podzielić? Ale to tylko mały nitpick.